src/olympia/users/admin.py

```diff
@@ -242,7 +242,7 @@ def get_search_results(self, request, queryset, search_term):
             # FilteredRelation we can force it...
             annotations = {
                 'activity_ips': GroupConcat(
-                    'activitylog__iplog__ip_address_binary',
+                    Inet6Ntoa('activitylog__iplog__ip_address_binary'),
                     distinct=True,
                     # You can't have multiple ip addresses in an IPv[4|6]Address object
                     # so we store the binary values in a CharField instead.
```

Comment:
this comment needs updating because we're no longer storing binary values.  (the `output_field` kwarg is potentially redundant too if `Inet6Ntoa` specifies an `output_field` but dunno)